libraries/riseUtils/index.js

```diff
@@ -111,50 +212,39 @@ export function generateBidsParams(validBidRequests, bidderRequest) {
 
 export function generateBidParameters(bid, bidderRequest) {
   const { params } = bid;
-  const mediaType = isBanner(bid) ? BANNER : VIDEO;
-  const sizesArray = getSizesArray(bid, mediaType);
+  const mediaTypes = getBidRequestMediaTypes(bid);
 
   if (isNaN(params.floorPrice)) {
     params.floorPrice = 0;
   }
 
   const bidObject = {
-    mediaType,
+    mediaType: mediaTypes.join(','),
     adUnitCode: getBidIdParameter('adUnitCode', bid),
-    sizes: sizesArray,
-    floorPrice: Math.max(getFloor(bid, mediaType), params.floorPrice),
+    sizes: getSizesArray(bid),
+    floorPrice: Math.max(getFloor(bid), params.floorPrice),
     bidId: getBidIdParameter('bidId', bid),
     loop: bid.bidderRequestsCount || 0,
     bidderRequestId: getBidIdParameter('bidderRequestId', bid),
     transactionId: bid.ortb2Imp?.ext?.tid || '',
     coppa: 0,
   };
 
-  const pos = deepAccess(bid, `mediaTypes.${mediaType}.pos`);
+  const pos = getPos(bid)
   if (pos) {
-    bidObject.pos = pos;
+    bidObject.pos = pos
```

Comment:
pos can be zero value, fix the if condition to pass also pos=0 (unknown position)